nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/main/java/org/apache/nifi/properties/sensitive/ProtectedNiFiProperties.java

```diff
@@ -459,46 +435,8 @@ public static int countProtectedProperties(NiFiProperties plainProperties) {
      * @param plainProperties the instance to count sensitive properties
      * @return the number of sensitive properties
      */
-    public static int countSensitiveProperties(NiFiProperties plainProperties) {
```

Comment:
Please restore this functionality and include it in the `#toString()` for visibility into how many _sensitive_ properties are actually _protected_.

Reply:
The method was just moved during refactoring, but the invocation was removed from `#toString()` so please restore that.

Reply:
The previous `toString` included a list of the sensitive providers.  Since we're not maintaining any references to providers (other than the default), I added the count of sensitive properties back into the string, e.g., `ProtectedNiFiProperties instance with 83 properties (3 protected and 6 sensitive)`